### PR TITLE
Completed Binary-Search-1 || Please Review

### PR DESCRIPTION
**Pull request by soumil100**

### CODE REVIEW

File: Problem2.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
+// Time Complexity :O(log n)
+// Space Complexity :O(1) ~ Please confirm
+//      since we are not storing any varibles
```
Comment (AbhijeetDhillon) [inline Problem2.java:3]: It's correct, and your reason it perfectly right!

**Comment by AbhijeetDhillon** [comment]
Nice work Soumil!

**Comment by AbhijeetDhillon** [comment]
Nice work Soumil!

**Comment by soumil100** [comment]
Thanks!

**Pull request closed without merging**